.travis.yml

```diff
@@ -4,8 +4,17 @@
 
 osx_image: xcode9.2
 language: objective-c
-cache: cocoapods
+cache:
+  - cocoapods
+  - bundler
+  - npm
```

Comment:
I dont think thsi actually works - i think you need to use:
```
  directories:
  - node_modules
```

Reply:
What makes you think it doesn't?

Reply:
See https://docs.travis-ci.com/user/caching/#Configuration

```yaml
language: ruby
cache:
  bundler: true
  directories:
  - node_modules # NPM packages
```

The example in there doesn;t have `npm` as a identifier for the `cache` option, it specifies it as a directory